src/2_RTB/PianoDiProgetto/sections/AnalisiDeiRischi.typ

```diff
@@ -0,0 +1,104 @@
+
+=== *RT2 - Difficoltà nell'adozione degli strumenti di gestione del progetto*
+Il team potrebbe incontrare sfide nell'apprendere l'utilizzo e nell'integrazione dei nuovi strumenti di gestione del progetto, causando ritardi nell'effettiva implementazione di un #glossary("Way of working") efficiente.
+  - *Identificazione*: la difficoltà nell'adozione degli strumenti può manifestarsi quando i membri del team incontrano dubbi o incertezze riguardo alle procedure da seguire o all'utilizzo degli strumenti stessi.
+  - *Valutazione*: Pericolosità: media; Occorrenza: bassa.
+  - *Prevenzione*:
+    - Norme di Progetto: tutti i dettagli del _way of working_ vengono concordati durante i meeting interni e normati all'interno delle Norme di Progetto. In caso di dubbi o incertezze, il riferimento a queste norme o la richiesta di chiarimenti all'Amministratore del progetto rappresentano un'importante risorsa per la risoluzione tempestiva di problemi legati agli strumenti e alle procedure;
+    - Inclusione di dubbi nell'Ordine del giorno: l'inclusione dei dubbi e delle incertezze nelle proposte dell'ordine del giorno del meeting interno successivo consente al team di affrontare in modo strutturato e tempestivo le difficoltà riscontrate, garantendo un ambiente in cui tali problemi possano essere risolti in modo collaborativo e efficiente.
+
+== Rischi di Comunicazione
+=== *RC1 - Mancata organizzazione della comunicazione interna*
+La mancata organizzazione della comunicazione interna si riferisce a situazioni in cui non esiste una struttura definita o linee guida chiare per la trasmissione efficace delle informazioni all'interno del team. Questa mancanza di direzione può portare a fraintendimenti, confusione o ritardi nella comunicazione.
+  - *Identificazione*: questo rischio si manifesta quando non esiste una struttura chiara per la comunicazione all'interno del team, può essere riconosciuto attraverso il sovrapporsi di messaggi, discussioni disordinate o mancanza di canali dedicati.
+  - *Valutazione*: Pericolosità: bassa; Occorrenza: bassa.
+  - *Prevenzione*:
+    - Pianificazione della comunicazione: stabilire canali di comunicazione specifici per diversi tipi di messaggi (ad esempio discussioni generali, questioni urgenti, cambiamenti dello stato delle _issue_ nell'ITS, proposte per l'ordine del giorno e altro);
+    - Ordine del giorno: definire un ordine del giorno chiaro per i meeting interni settimanali, con tempi stabiliti per ciascun argomento;
+    - Stand-up meeting giornalieri: riunioni brevi e quotidiane favoriscono una migliore comprensione dello stato del progetto, consentendo a ciascun membro del team di aggiornare gli altri sui progressi, sulle sfide incontrate e sui prossimi passi. Questa pratica aiuta a identificare i problemi in modo tempestivo, a fornire supporto reciproco e a mantenere il team allineato con gli obiettivi comuni.
+
+=== *RC2 - Esacerbazione di conflitti interni*
```

Comment:
Esacerbacosa?

Reply:
https://www.treccani.it/vocabolario/esacerbazione/

Reply:
Fair enough